testsuite/main.go

```diff
@@ -47,7 +47,7 @@ func main() {
 	serviceImageArg := flag.String(
 		"service-image",
 		"",
-		"Name of Docker image that will be used to launch service containers")
+		"Name of Nginx Docker image that will be used to launch service containers")
```

Comment:
why specify that this is Nginx here?

Reply:
oh now i see. just to clarify my understanding: we're renaming the example to nginx to make it explicit what the thing comes with